packages/create-svelte-ux/bin.js

```diff
@@ -107,6 +107,15 @@ const template = options_cli.template
 if (p.isCancel(template)) {
   process.exit(1);
 }
+const svelteNext = await p.confirm({
+  message: 'Would you like to use Svelte 5 (unstable)?',
+  initialValue: false,
+});
+if (p.isCancel(svelteNext)) {
+  process.exit(1);
+}
+const svelteVersion = svelteNext ? '^5.0.0-next.148' : '^4.2.17';
```

Comment:
I think `svelte@latest` would be good if it would give Svelte 5.
Though a `@latest` will give the latest version that is not a pre-release.

Part of the post in the link above:
> You can select the 'Try the Svelte 5 preview' option when starting a new SvelteKit project.

Reply:
Ah I see what we can do here.
`npm i -D svelte@next` installs the pre. `npm i -D svelte@latest` would currently give `"svelte": "^4.2.7"`. 

Maybe updating both specifiers?